### DISCUSSION TITLE
Trying to compile connected to more than 5 Databases

### DISCUSSION THREAD

**Discussion by pgrsman** [Riverside-Software/pct#472] [Q&A] [answered]
Has anyone successfully compiled connected to more than 5 databases (the Progress default)?  I've been trying different methods of setting -h inside of PCTCompile without success.

Thanks

**Comment by bulagump** [comment]
Which version?  I have used it connected to many more than five databases at the same time, but I haven't used the utility for over six months, so that was likely a prior version than what is currently out there.

**Comment by pgrsman** [comment, marked as answer]
UPDATE: I solved my own problem with this little gem under PCTCompile:

            <Option name="-h" value="10" />
